x-pack/plugins/enterprise_search/public/applications/app_search/components/document_creation/document_creation_logic.ts

```diff
@@ -33,6 +33,8 @@ interface DocumentCreationValues {
   warnings: string[];
   errors: string[];
   summary: DocumentCreationSummary;
+  availableElasticsearchIndices: object[]; // TODO common Elasticsearchindex type
```

Comment:
Do we have a common Elasticsearchindex Type that I can pick partials out of it according to backend data ?